pygameMenu/menu.py

```diff
@@ -1175,42 +1159,38 @@ def _handle_joy_event(self):
         if self._joy_event & _JOY_EVENT_RIGHT:
             self._right()
 
-    def _main(self, events=None):
+    def update(self, events=None):
         """
-        Main function of the loop.
+        Update the status of the menu using external events.
 
         :param events: Pygame events
         :type events: list
         :return: True if mainloop must be stopped
         :rtype: bool
         """
+        assert isinstance(events, (list, type(None)))
+
         break_mainloop = False
         if events is None:

```

Comment:
the event should be get in the `mainloop` function
it shall not be optional in `update` one (as for widgets)

Reply:
true!